File: core/src/main/java/dev/bee/kanjianki/core/StudyLayoutPolicy.java
package dev.bee.kanjianki.core;

public final class StudyLayoutPolicy {
    private StudyLayoutPolicy() {
    }

    public static int writingPadHeightDp(int screenHeightDp) {
        if (screenHeightDp < 700) {
            return 300;
        }
        if (screenHeightDp < 820) {
            return 340;
        }
        return 390;
    }
}

File: core/src/main/kotlin/dev/bee/kanjianki/core/StudyLayoutPolicy.kt
package dev.bee.kanjianki.core

object StudyLayoutPolicy {
    @JvmStatic
    fun writingPadHeightDp(screenHeightDp: Int): Int {
        if (screenHeightDp < 700) {
            return 300
        }
        if (screenHeightDp < 820) {
            return 340
        }
        return 390
    }
}
